sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/InstancesOperationError.cs

```diff
@@ -6,13 +6,13 @@
 namespace Azure.Iot.TimeSeriesInsights
 {
     /// <summary>
-    /// A particular API error with an error code and a message.
+    /// Time Series Insights operation error with an error code and a message.
     /// </summary>
     [CodeGenModel("TsiErrorBody")]
-    public partial class InstancesOperationError
+    public partial class TimeSeriesOperationError
```

Comment:
Renaming since this is used for other operations such as types, hierarchies and not only instances.